libpod/image.go

```diff
@@ -60,6 +75,7 @@ func (r *Runtime) PullImage(imgName string, allTags bool, reportWriter io.Writer
 	var (
 		images []string
 	)
+	sc := common.GetSystemContext("")
```

Comment:
We have a SignaturePolicyPath in the Runtime's config - if the one the user passes in is empty, we should use that one